Dockerfile.ppc64le

```diff
@@ -95,25 +94,12 @@ RUN set -x \
 
 
 # Install Go
-# ppc64le doesn't have official go binaries, so use the version of go installed from the image
-# to build go from source.
-# NOTE: ppc64le has compatibility issues with older versions of go, so make sure the version >= 1.6
+# NOTE: official ppc64le go binaries weren't available until go 1.6.4 and 1.7.4
```

Comment:
@thaJeztah how about this? I want to keep something so it's obvious that earlier versions of go will break this build. Also I removed the comments from contrib because they really didn't need to be there.